api-reference/beta/resources/alert.md

```diff
@@ -0,0 +1,120 @@
+# alert resource type
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Alerts represent potential security issues within a customer's tenant that have been identified by Microsoft or partner security solutions. Alerts can be used to unify and streamline alert management across all integrated solutions. See sample queries in [Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer).
+
+For Public Preview (April 2018), alerts can be retrieved from the following Providers: **Azure Security Center**, and **Azure Active Directory Identity Protection**. Additional Alert Providers will integrate in the following months.
+
+## Methods
+
+| Method   | Return Type|Description|
+|:---------------|:--------|:----------|
+|[Get alert](../api/alert_get.md) | [alert](alert.md) |Read properties and relationships of alert object.|
+|[Update](../api/alert_update.md) | [alert](alert.md) |Update alert object. |
+|[List alerts](../api/alert_list.md) | [alert](alert.md) collection |Get a alert object collection.|
+
+## Properties
+
+### ( \* = Mandatory Field )

```

Comment:
Remove this H3 section that mentions the mandatory field. Just add "\* Indicates a mandatory field." at the end of the table.